hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieActiveTimeline.java

```diff
@@ -156,6 +157,18 @@ public void createNewInstant(HoodieInstant instant) {
     createFileInMetaPath(instant.getFileName(), Option.empty(), false);
   }
 
+  public void createRequestedReplaceCommit(String instantTime, String actionType) {
+    try {
+      HoodieInstant instant = new HoodieInstant(State.REQUESTED, actionType, instantTime);
+      LOG.info("Creating a new instant " + instant);
```

Comment:
Is this info log necessary?

Reply:
It's not necessary, just to be consistent with the previous method `createNewInstant`